### ISSUE TITLE
Navigation links are pushed down in header

### ISSUE THREAD

**Issue by cpharris** [godaddy/wp-velux-theme#56]
Hello,

The logo is pushing the navigation links below the "site-title-wrapper" div.

I've tried adding and remove the Title and Tag Line but it doesn't seem to make a difference.

You can see it here: [website example](http://www.chadpharris.com)

And here's a screenshot from Chrome.

![image](https://user-images.githubusercontent.com/2033082/35101827-eb5e1906-fc2e-11e7-8bb7-652c74de7116.png)

**Comment by EvanHerman** [comment]
Hi @cpharris 

I'll do some testing to see if this is something occurring out of the box, or if there is a conflict with a plugin you have installed.

In the meantime, you should be able to fix the issue by dropping the following CSS snippet into the 'Additional CSS' section of the customizer.

```css
.site-title-wrapper {
   width: auto;
}
```

I will report back here with some findings when I've had a moment to test.


Evan

**Comment by cpharris** [comment]
Excellent! Thank you Evan.

I added the width:auto CSS for now.

**Comment by EvanHerman** [comment]
Hi @cpharris 

Glad to hear that worked for you! I did go ahead and do some testing and noticed the same issue on a fresh install with nothing else active.

I've gone ahead and issued a PR against our development branch so we can get this fix included in the next release of Velux.

Thanks so much for reporting the issue here. I'll leave this issue open so it doesn't fall through the cracks and so anyone else facing a similar issue can make note of the fix I've provided. When we get v1.1.2 pushed out, this fix will be included and the above CSS snippet will no longer be necessary.


Evan
